configurator/configurator.go

```diff
@@ -74,6 +66,15 @@ func addConfigFile() {
 }
 
 func setDefaultConf() {
+	var defaultStartPath = "."
+	var defaultBlackList = ExtList{"lnk", "ini2", "bin", "tmp"}
+	var defaultWhiteListDocs = ExtList{"doc", "docx", "xls", "xlsx", "ppt", "pptx", "pdf"}
+	var defaultWhiteListImgs = ExtList{"png", "jpg", "jpeg", "raw"}
+	var defaultWhiteListExts = slices.Concat(defaultWhiteListDocs, defaultWhiteListImgs)
+	var defaultBlackListFiles = FileList{"~.ini2"}
+	var defaultSizeLimit int64 = 5_000_000
+	var defaultContentBlacklist = []string{"powershell"}
```

Comment:
BTW you may also use here the same syntax for batch declaration that was used on package level.
```
var (
  defaultStartPath = ...
  defaultBlackList = ...
)
```